contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -4205,4 +4205,98 @@ LANGUAGE plpgsql STABLE;
 CREATE OR REPLACE FUNCTION sys.bbf_pivot()
 RETURNS setof record
 AS 'babelfishpg_tsql', 'bbf_pivot'
-LANGUAGE C STABLE;
\ No newline at end of file
+LANGUAGE C STABLE;
+
+-- wrapper functions for reverse
+CREATE OR REPLACE FUNCTION sys.reverse(string ANYELEMENT)
+RETURNS sys.VARCHAR
```

Comment:
How are we making sure that for any datatype (it can be varchar, nvarchar, text etc) this function is not getting picked up? I think the return datatype kind of serves that purpose in the test cases.

Reply:
kindly refer here: https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/2719#discussion_r1683799427 to understand how expected function is getting called. Also to test this we can check the expected output of the tests added in reverse-vu-verify with different datatype, there return data type of the output verify that expected function definition is getting picked.